plugins/importtext_procedure.cc

```diff
@@ -689,14 +691,36 @@ struct ImportTextProcedureWorkInstance
                 };
 
             if (config.headers.empty()) {
+
+                vector<string> fields;
+
                 // Read header line
-                std::getline(stream, header);
+                string prevHeader;
                 lineOffset += 1;
-                ML::Parse_Context pcontext(filename,
-                                           header.c_str(), header.length(), 1, 0);
+                while(true) {
+                    std::getline(stream, header);
+
+                    if(!prevHeader.empty()) {
+                        prevHeader += header;
+                        header.assign(std::move(prevHeader));
+                    }
 
-                vector<string> fields
-                    = ML::expect_csv_row(pcontext, -1, separator);
+                    try {
+                        ML::Parse_Context pcontext(filename,
+                                               header.c_str(), header.length(), 1, 0);
+                        fields = ML::expect_csv_row(pcontext, -1, separator);
+                        break;
+                    }
+                    catch (std::exception & exp) {
+                        if(config.allowMultiLines &&
```

Comment:
This is risky.  Someone changing the error message will never think that this could be breaking some functionality.  Can we use a specific Exception type or even better can we change expect_csv_row to return a "line continuation" type of code?